YGOProAnalyticsServer/EventHandlers/YgoProAnalysisBasedOnDataFromYgoProServer.cs

```diff
@@ -291,5 +312,27 @@ private void _analyzeBanlist(DuelLog duelLog)
 
             banlistStatistics.IncrementHowManyTimesWasUsed();
         }
+
+        private async Task _addPlayableBanlistsToDecklist(Decklist decklist, IEnumerable<Banlist> banlists)
+        {
+
+            foreach (var banlist in banlists)
+            {
+                if (!decklist.PlayableOnBanlists.Contains(banlist) && _banlistService.CanDeckBeUsedOnGivenBanlist(decklist, banlist))
```

Comment:
Currently PlayableOnBanlists is never loaded from db. 
![image](https://user-images.githubusercontent.com/38086920/60061876-18487100-96f7-11e9-88ab-ab1bbed39895.png)